z_infra_provisioning/cloud_infra_center/ocp_upi/roles/configure-install-config/tasks/main.yaml

```diff
@@ -10,6 +10,28 @@
 ---
 # tasks file for configure-install-config
 
+- name: Create install-config.yaml
+  template:
+    src:  templates/install-config-yaml.j2
+    dest: "install-config.yaml"
+
+- name: Split the cluster domain name
+  set_fact: 
+    clustername: "{{ cluster_domain_name.split('.')[0] }}"
+    basedomain_one: "{{ cluster_domain_name.split('.')[1] }}"
+    basedomain_two: "{{ cluster_domain_name.split('.')[2] }}"
```

Comment:
What if the cluster domain name is `example.beijing.china.com`

Reply:
good catch, I forget it, I will add this format later.

Reply:
Fix it by below testing:
Code:
```
    basedomain: "openshift.example.com"
    basedomain2: "example.ibm.test.com"
    basedomain3: "test.example.icic-com"
   
  tasks:
  - name: Print1
    ansible.builtin.debug:
      msg: "basedomain: {{ basedomain }} , basedomain1: {{ basedomain.split('.')[1:] | join('.') | string  }}"

  - name: Print2
    ansible.builtin.debug:
      msg: "basedomain2: {{ basedomain2 }},  basedomain21: {{ basedomain2.split('.')[1:] | join('.') | string  }}"

  - name: Print3
    ansible.builtin.debug:
      msg: "basedomain3: {{ basedomain3 }} ,  basedomain31: {{ basedomain3.split('.')[1:] | join('.') | string  }}"
```
Testing:
```
TASK [Print1] *************************************************************************************************************************************************
ok: [127.0.0.1] => {
    "msg": "basedomain: openshift.example.com , basedomain1: example.com"
}

TASK [Print2] *************************************************************************************************************************************************
ok: [127.0.0.1] => {
    "msg": "basedomain2: example.ibm.test.com,  basedomain21: ibm.test.com"
}

TASK [Print3] *************************************************************************************************************************************************
ok: [127.0.0.1] => {
    "msg": "basedomain3: test.example.icic-com ,  basedomain31: example.icic-com"
}

```

Reply:
This is not an one-size-fits-all solution, what if cluster domain name is `example.1.2.beijing.china.com`